readme.txt
-------------------------------------------------------
    Thank you for downloading Impactshirts Theme!
-------------------------------------------------------
This and any other Theme provided by CyberChimps should
be free of bugs, however do note that we're only humans
and if you spot a bug do us a favor and report here
http://cyberchimps.com/forum/free/impactshirts/

If something goes wrong, or you need our assistance with anything
just register and post your question(s).

-------------------------------------------------------
    Impactshirts Features:
-------------------------------------------------------

- Theme Options
From here you can customize your above the fold
section such as your headline, sub-headline and content.
Also your call to action button text and URL. As well as
an image on the right. Image can be just that or even YouTube
or any other video too.

- Widgets
Well you're in luck my friend, Impactshirts offers 9 of them.
And yes 3 boxes in your home.php are widgets as well.

- Page Templates
There is plenty of that. 9 Templates are very much self-explanatory
once you're on done with the content on your page, go to Page Attributes
on the right and choose any available Template.

We Have:
Blog (full posts), Blog Summaries, Content/Sidebar, Sidebar/Content,
Content/Sidebar Half, Sidebar/Content Half, Full Width Page, even a
landing page in case that you don't want anything else but the clean
white design and your logo, perfect for PPC or anything else. There's
also a sitemap Template.

- Menus
Oh boy, we love menus and that's why we decided to give you little bit
more than what you're probably use to. Impactshirts offers 4 of them
One at very top, second in header, third just after the header, we like
to call that sub-header menu and the fourth one is all the way down in
footer area.

What's great about this is that the other 3 are kind of on-demand,
activate when needed, if not oh well...

-------------------------------------------------------
    License:
-------------------------------------------------------

Yes I know, but we have to say few words about
that.

The Theme itself is nothing but 100% GPL3.
See license.txt for further details. There
are however some parts of this Theme which are not
GPL, however they are GPL-Compatible and thats still
good.

See headers of JS files for further details.

-------------------------------------------------------
    Adopting Impactshirts Theme (Theme Re-Distribution):
-------------------------------------------------------

It's GPL so yes you can base your Themes from Impactshirts
please note that if you do the GPLv3 License needs to stay
as well as all copyright notices included in all parts of
this Theme as well as third-party add-ons as listed above.

All derivative Themes must be much different from the original
work, which includes the Theme design as well as all the codes
from Impactshirts Theme.


changelog.txt
Changelog Legend:
----

[+] = Added
[*] = Changed
[^] = Moved
[=] = No Changes
[x] = Deleted
[!] = Bugs

(02/26/16) - Improvements - Version 1.9.7.9

[+] - Added an option to customize copyright text in footer.

----

(02/08/16) - Improvements - Version 1.9.7.8

[!] - Fixed CSS validation errors


----

(07/17/15) - Improvements - Version 1.9.7.7

[+] - Added options to customizer as per wordpress.org guidelines
[+] - Added new plugin suggestion.


----

(02/04/15) - Improvements - Version 1.9.7.5
[+] - Added new plugin suggestion.
[!] - Fix on turkish translation.

(10/05/14) - Improvements - Version 1.9.7.5

[*] Minify all backend scripts and styles
[*] Improve comply to coding standards
[*] Run css through csscomb

----

(09/20/14) - Improvements - Version 1.9.7.4

[+] Added `get_post_type()` to all `get_template_part()` so that CPT can have a differnt template in the child theme.
[*] Improve comply to coding standards
[*] Update Translations

----

(09/06/14) - Improvements - Version 1.9.7.3

[+] Add iFeatured Slider as recommended plugin
[*] Escape variables to improve security
[*] Small code improvements

----

(08/24/14) - Improvements - Version 1.9.7.2

[!] Fix stray p tag in activation notice
[+] Add wrapper for Breadcrumb NavXT
[+] Define content width depending on page template

----

(08/10/14) - Improvements - Version 1.9.7.1

[*] Improve the organisation of the functions
[*] Clean up code
[!] Fix screen reader link
[+] Add filter `impactshirts_number_footer_widgets` to change when the `fit` class is added in the footer widgets.

----

(07/25/14) - Improvements - Version 1.9.7.0

[!] Improve Impactshirts Grid
[x] Remove default logo as per WordPress.org guidelines
[*] Update Screenshot

----

(07/18/14) - Improvements - Version 1.9.6.9

[!] Fix very long submenu items
[!] Fix path to ie7-fix.gif
[!] Fix front page error message

----

(07/11/14) - Improvements - Version 1.9.6.8

[*] Add taxonomy description to archive page
[*] Seperate featured area from front page

----

(07/04/14) - Improvements - Version 1.9.6.7

[*] Update cs_CZ translation
[*] Small code style improvements

----

(06/27/14) - Improvements - Version 1.9.6.6

[!] Update NL translation to fix theme options bug
[*] Improve page template switch from Impactshirts v2.0
[*] Enclose add_image_size within a function

----

(06/20/14) - Improvements - Version 1.9.6.4-5

[+] Change page templates when reverting from Impactshirts v2.0
[*] Move tags under categories under posts

----

(06/11/14) - Improvements - Version 1.9.6.3

[*] Fix minified js

----

(06/06/14) - Improvements - Version 1.9.6.2

[*] Updated Hindi Translation
[*] Updated FitVids.js in minified version
[*] Changed some codestyling

----

(05/30/14) - Improvements - Version 1.9.6.1

[*] Updated Ukrainian, Bulgarian, German, French(Canada), Hindi and Dutch Translation
[*] Updated FitVids.js in development version

----

(05/23/14) - Improvements - Version 1.9.6.0

[*] Updated Hungarian Translation
[+] Added new Canadian French Translation

----

(05/16/14) - Improvements - Version 1.9.5.9

[*] Updated new landing page for Impactshirts 2.o
[*] Updated Danish Translation

----

(05/09/14) - Improvements - Version 1.9.5.8

[*] Readded default logo
[*] Updated translations

----

(04/30/14) - Improvements - Version 1.9.5.7

[+] Added new recommended Plugin
[*] Updated logo
[*] Changed some code styling
[!] Fixed translations files

----

(04/22/14) - Improvements - Version 1.9.5.6

[*] Update POT File
[*] Update translations
[*] Improve strings for i18n
[!] Fixed wp_editor issue with WordPress 3.9

----

(04/16/14) - Improvements - Version 1.9.5.5

[*] Change post layout selection from radio buttons to dropdown menu
[!] Fixed "Default" showing twice on post layout

----

(04/08/14) - Improvements - Version 1.9.5.4

[*] Updated POT file
[*] Added logo to featured image
[!] Fixed check if Yoast SEO breadcrumbs activated
[!] Fixed image links in admin area

----

(03/14/14) - Improvements - Version 1.9.5.3

[*] Updated other cyberchimps page
[^] Moved theme options to own folder
[!] Fixed loading correct sidebar when using post templates

----

(02/25/14) - Improvements - Version 1.9.5.2

[+] Added admin notice for when Impactshirts 2.0 releases
[+] Added missing text domain
[+] Added skip link
[*] Changed widget menu class to WordPress standards
[*] Tweeked "Posted on" time info
[*] Updated screenshot
[*] Updated translations

----

(02/13/14) - Improvements - Version 1.9.5.1

[!] Fixed widget bug

----

(02/09/14) - Improvements - Version 1.9.5.0

[+] Added support for minified css version
[*] Restructured CSS files - moved styles to core/css/
[*] Dispaly plugin install notification only on theme.php
[*] Customized theme activation notice
[^] Moved sidebar functions in functions-sidebar.php

----

(02/02/14) - Improvements - Version 1.9.4.9

[+] Updated to new social icon function

----

(01/28/14) - Improvements - Version 1.9.4.7-8

[*] Allow for gallery image size can be defined
[!] Fixed disabling breadcrumbs

----

(01/19/14) - Improvements - Version 1.9.4.6

[*] Improved Impactshirts Breadcrumbs
[+] Added support for Breadcrumb NavXT, Breadcrum Trail and Yoast SEO
breadcrumbs.

----

(01/12/14) - Improvements - Version 1.9.4.5

[*] Made Theme Options impactshirts
[*] Update coyright to 2014
[x] Removed old code in header.php

----

(01/07/14) - Improvements - Version 1.9.4.4

[+] Added improved support gallery images
[x] Removed old tags

----

(12/22/13) - Improvements - Version 1.9.4.3

[+] Allow switching minified scripts with SCRIPT_DEBUG
[x] Removed code for Backward Compatibility of WP 3.4
[*] Improved impactshirts_widget_menu_class as suggested by markoheijnen
[*] Allow Stylesheet version update automatically.
[!] Fix disabling breadcrumbs

----

(12/15/13) - Improvements - Version 1.9.4.2

[+] Made search form accessible
[*] Updated screenshot to 880x660px
[*] Moved breadcrumb check from template to functions.php
[*] Updated impactshirts.pot

search.php
<?php

// Exit if accessed directly
if( !defined( 'ABSPATH' ) ) {
	exit;
}

/**
 * Search Template
 *
 *
 * @file           search.php
 * @package        Responsive
 * @author         Emil Uzelac
 * @copyright      2003 - 2014 CyberChimps
 * @license        license.txt
 * @version        Release: 1.0
 * @filesource     wp-content/themes/responsive/search.php
 * @link           http://codex.wordpress.org/Theme_Development#Search_Results_.28search.php.29
 * @since          available since Release 1.0
 */

get_header(); ?>



	<?php 
	
	global $query_string;
	$s_results = query_posts( $query_string . '&posts_per_page=20' );
	if( get_current_user_id() == 1 && false ){
		//restore_current_blog();
		$s_query = $query_string . '&posts_per_page=20&post_type=any';
		/*$q1 = get_posts(array(
			'post_type' => 'any',
			's' => get_search_query()
		));
		$q2 = get_posts(array(
			'post_type' => 'any',
			'meta_query' => array(
				array(
					'key' => 'ncDesignNumber',
					'value' => get_search_query(),
					'compare' => 'LIKE'
				)
			)
		));
		$s_results = array_merge( $q1, $q2 );*/

		echo '<br><br><br> start <br><br><br>';
		//$s_results = get_posts( $s_query );
		global $wpdb; echo $wpdb->posts;
		i_print( $s_results );
	}
	
	if( have_posts() ) : ?>


	<div class="full_image_outer">
    	<div class="full_image_inner">
        	<div class="grid col-940 nomargin">
        		<h2 class="full_image_title">Showing search results for <b><?php echo get_search_query(); ?></b></h2>
            </div>
            <div class="grid col-940 search_recs">
                <div class="rec_simpler">
                    Refine your search.
                </div>
            </div>
            <div class="clearfix"></div>
            <div class="search_page_form">
            	<div class="widget-wrapper"><?php get_search_form(); ?></div>
            </div>
            <?php display_search_terms(); ?>
            <div class="clearfix"></div>
        </div>
    </div>

<?php
		echo '<div id="wrapper" class="clearfix">';
		echo '<div id="content-search" class="grid col-940" style="text-align:center;">';
 
		// get_template_part( 'loop-header' ); ?>

				<div class="cat-gallery">
					<ul>
						<?php 
							$post_counter = 0;
							while( have_posts() ) : 
							the_post(); 
							$post_counter++; 
							$featuredImage = get_the_post_thumbnail();
							$image = wp_get_attachment_image_src( get_post_thumbnail_id( $post->ID ), 'large' );
							$itemNumber = get_post_meta($post->ID, 'ncDesignNumber', true);
						?>
						<li class="grid col-172 <?php if( $post_counter == 5 ){ echo 'fit'; $post_counter = 0; } else { } ?>">
							<ul>

								<li>
									<a href="<?php the_permalink() ?>"><?php 
									if ( has_post_thumbnail() ) : 
									  echo '<img class="attachment-post-thumbnail wp-post-image" src="'.$image[0].'" title="'.get_the_title().'" alt="'.get_the_title().'" height="'.$image[2].'" width="'.$image[1].'"/>';
									  echo '<div class="hoverLabel">VIEW</div>';
									endif; 
									?></a>
								</li>
								<li class="item">
                                <a href="<?php the_permalink() ?>" title="<?php the_title(); ?>">
                                <?php 
                                    $title = get_the_title();
                                    $title = strlen($title) > 23 ? substr($title,0,23) : $title;
                                    $title = ucwords($title);
                                ?>
                                <ul>
                                <li class="item-view" style="width:77%;"><p><?php echo $title; ?></p></li>
                                <li class="item-number" style="width:25%"><p><?php echo $itemNumber ?></p></li>
                                </ul>
                                </a>
								</li>
							</ul>
						</li>
						<?php endwhile; ?>
					</ul>
				</div>
        
        <?php
        $args = array(
			'end_size' 	=> 0,
			'mid_size' 	=>5,
			'type'		=> 'list'
		);

		echo paginate_links( $args );

		echo '</div>';
		echo '</div>';

	// If we don't have any matches
	else : ?>

	<div class="full_image_outer">
    	<div class="full_image_inner">
        	<div class="grid col-940 nomargin">
        		<h2 class="full_image_title">Sorry, we couldn't find any matches for <b><?php echo get_search_query(); ?></b></h2>
            </div>
            <div class="grid col-940 search_recs">
                <div class="rec_spelling">
             		Make sure the spelling is correct.
                </div>
                <div class="rec_simpler">
                    Try using a simpler search.
                </div>
            </div>
            <div class="clearfix"></div>
            <div class="search_page_form">
            	<div class="widget-wrapper"><?php get_search_form(); ?></div>
            </div>
            <?php display_search_terms(); ?>
            <div class="clearfix"></div>
        </div>
    </div>
    
    
    
    <div id="wrapper" class="clearfix">
    <div id="content-search" class="grid col-940" style="text-align:center;">
    	
<?php
		// get_template_part( 'loop-no-posts' );

		$taxonomy     = 'design-category';
		$orderby      = 'name';
		$show_count   = 0;      // 1 for yes, 0 for no
		$pad_counts   = 0;      // 1 for yes, 0 for no
		$hierarchical = 1;      // 1 for yes, 0 for no
		$title        = '';
		$walker		  = new Walker_Category_Find_Parents();

		$args = array(
		  'taxonomy'     => $taxonomy,
		  'orderby'      => $orderby,
		  'show_count'   => $show_count,
		  'pad_counts'   => $pad_counts,
		  'hierarchical' => $hierarchical,
		  'title_li'     => $title,
		  'walker'		 => $walker
		);
		?>
		<div class="cat-list">
        	<h3>Popular Design Categories</h3>
			<ul>
			<?php wp_list_categories( $args ); ?>
			</ul>
		</div> <?php

		$args = array( 
			'post_type' 	=> 'design',
			'orderby' 		=> 'date',
			'order'   		=> 'DESC', 
		);
		$q = new WP_Query( $args );

		if( $q->have_posts() ) : ?>
					<h3>Our Most Recent Designs</h3>
					<div class="cat-gallery">
						<ul>
							<?php 
								$post_counter = 0;
								while( $q->have_posts() ) : 
								$q->the_post(); 
								$post_counter++; 
								$featuredImage = get_the_post_thumbnail();
								$image = wp_get_attachment_image_src( get_post_thumbnail_id( $post->ID ), 'large' );
								$itemNumber = get_post_meta($post->ID, 'ncDesignNumber', true);
							?>
							<li class="grid col-172 <?php if( $post_counter == 5 ){ echo 'fit'; $post_counter = 0; } else { } ?>">
								<ul>
	
									<li>
										<a href="<?php the_permalink() ?>"><?php 
										if ( has_post_thumbnail() ) : 
										  echo '<img class="attachment-post-thumbnail wp-post-image" src="'.$image[0].'" title="'.get_the_title().'" alt="'.get_the_title().'" height="'.$image[2].'" width="'.$image[1].'"/>';
										  echo '<div class="hoverLabel">VIEW</div>';
										endif; 
										?></a>
									</li>
									<li class="item">
									<a href="<?php the_permalink() ?>" title="<?php the_title(); ?>">
									<?php 
										$title = get_the_title();
										$title = strlen($title) > 23 ? substr($title,0,23) : $title;
										$title = ucwords($title);
									?>
									<ul>
									<li class="item-view" style="width:77%;"><p><?php echo $title; ?></p></li>
									<li class="item-number" style="width:25%"><p><?php echo $itemNumber ?></p></li>
									</ul>
									</a>
									</li>
								</ul>
							</li>
							<?php endwhile; ?>
						</ul>
					</div>
                    </div>
	<?php
		wp_reset_query();
		endif;


	endif;
	?>

</div><!-- end of #content-search -->

<?php get_footer(); ?>
